pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -55,6 +55,15 @@ func (tc *TidbCluster) PDImage() string {
 	return image
 }
 
+func (tc *TidbCluster) PDVersion() string {
```

Comment:
The image and tag can also be configured in `tc.Spec.PD.Image`.

Reply:
A function [here](https://github.com/pingcap/tidb-operator/blob/master/pkg/backup/util/util.go#L415) was defined to get image tag from `PDImage()`.